html_writer.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+"""Quick and dirty HTML writer."""
+
+import re
+import sys
+
+HTML_SCRIPT_CODE = 'garden.js'
+
+LINK_RE = re.compile(r'https?://[a-zA-Z0-9./-]*')
+
+
+class HTMLWriter(object):
+
+  SPACE = '&nbsp;'
+
+  def __init__(self, out=sys.stdout):
+    self.out = out
+    self.in_row = False
+
+  def write(self, content):
+    self.out.write(content)
+
+  def nl(self):
+    self.out.write('<br/>')
+
+  def preamble(self, css):
+    self.write('<!DOCTYPE html>\n<html lang="en">')
```

Comment:
Hmm.. let's not build a new static site generation / templating framework and use something battle tested like http://flask.pocoo.org/?

Reply:
We could use flask, since we already import that internally. I'm slightly hesitant to take a dependency right now, before we figure out how we want to run this. Can we defer that discussion to the next DevExp meeting?

Reply:
Lets do that. We should think about how we want to build this as a web site/service.